src/components/ClickWrapper/index.jsx

```diff
@@ -129,9 +129,16 @@ export default ClickWrapper;
 
 
 ClickWrapper.propTypes = {
+  /** Determines whether width should be set to 100%. **/
   full: t.bool,
+  /** Nested structure that needs this wrapper **/
   children: t.node.isRequired,
+  /** URL that should be added to the click action.
```

Comment:
'This should be the URL to where the user will be directed when clicking on the link. Alternatively, if a function is passed that function will be executed when the user clicks on the link'

Reply:
Updated